### DISCUSSION TITLE
I don't find the property "type" in MikroOrmModuleSyncOptions

### DISCUSSION THREAD

**Discussion by mrZedov** [mikro-orm/nestjs#158] [Q&A]
I don't find the property "type" in MikroOrmModuleSyncOptions
how should I indicate that I want to use PostgreSQL? )
I just upadeted MikroORM to 6.0.3 and @mikro-orm/nestjs to 5.2.3
before it was there and everything worked.

I use it in MikroOrmModule.forRoot(.)

thanks )

**Comment by dungkaka** [comment]
Well, I guess you are finding "driver" option that replace "type" in v6
